src/semeio/fmudesign/create_design.py

```diff
@@ -17,6 +17,36 @@
 from semeio.fmudesign.iman_conover import ImanConover
 
 
+def is_consistent_correlation_matrix(matrix):
+    # Convert to numpy array if not already
+    matrix = np.array(matrix)
+
+    # Check if matrix is 2D and square
```

Comment:
this one is a bit different than the others. this is a true show-stopper, whereas the others can be "fixed". since `nearest_correlation_matrix` raises on this too, I would drop it from this function.

Reply:
Thanks, I've removed it.